apps/web/pages/auth/verify-email.tsx

```diff
@@ -0,0 +1,60 @@
+import { MailOpenIcon } from "lucide-react";
+import { useSession } from "next-auth/react";
+import { useRouter } from "next/router";
+import { useEffect } from "react";
+
+import { APP_NAME } from "@calcom/lib/constants";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { trpc } from "@calcom/trpc";
+import useEmailVerifyCheck from "@calcom/trpc/react/hooks/useEmailVerifyCheck";
+import { Button, EmptyScreen, showToast } from "@calcom/ui";
+
+import PageWrapper from "@components/PageWrapper";
+
+function VerifyEmailPage() {
+  const { data } = useEmailVerifyCheck();
+  const { data: session } = useSession();
+  const router = useRouter();
+  const { t } = useLocale();
+  const mutation = trpc.viewer.auth.resendVerifyEmail.useMutation();
+
+  useEffect(() => {
+    if (data?.isVerified) {
+      router.replace("/getting-started");
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [data?.isVerified]);
```

Comment:
If you navigate back to this tab after verifying your account we redirect to straight to getting started

Reply:
Can we just return a new view that says. "Email is already verified"? Maybe add an CTA that takes you back to the dashboard.

Reply:
I dont think it was clear what i meant by this.

if you switch back to this tab after you have verified your email. It navigates you to getting-started. Meaning you dont have to log in or hard refresh